deploy/ansible/run.sh

```diff
@@ -16,20 +16,22 @@ wireguard_server_private_key=$WIREGUARD_SERVER_PRIVKEY
 elkElasticUser=$ELASTIC_USER
 elkElasticPassword=$ELASTIC_PASSWORD
 grafanaAdminPassword=$GRAFANA_ADMIN_PASSWORD
+
 NODE_01_ACCOUNTID=$NODE_01_ACCOUNTID
-NODE_01_VALIDATOR_PRIVKEY=$NODE_01_VALIDATOR_PRIVKEY
-NODE_01_P2PIDENTITYPRIVATEKEY=$NODE_01_P2PIDENTITYPRIVATEKEY
 NODE_02_ACCOUNTID=$NODE_02_ACCOUNTID
-NODE_02_VALIDATOR_PRIVKEY=$NODE_02_VALIDATOR_PRIVKEY
-NODE_02_P2PIDENTITYPRIVATEKEY=$NODE_02_P2PIDENTITYPRIVATEKEY
 NODE_03_ACCOUNTID=$NODE_03_ACCOUNTID
+
+NODE_01_VALIDATOR_PRIVKEY=$NODE_01_VALIDATOR_PRIVKEY
+NODE_02_VALIDATOR_PRIVKEY=$NODE_02_VALIDATOR_PRIVKEY
 NODE_03_VALIDATOR_PRIVKEY=$NODE_03_VALIDATOR_PRIVKEY
+
+NODE_01_P2PIDENTITYPRIVATEKEY=$NODE_01_P2PIDENTITYPRIVATEKEY
+NODE_02_P2PIDENTITYPRIVATEKEY=$NODE_02_P2PIDENTITYPRIVATEKEY
 NODE_03_P2PIDENTITYPRIVATEKEY=$NODE_03_P2PIDENTITYPRIVATEKEY
-NODE_04_ACCOUNTID=$NODE_04_ACCOUNTID
-NODE_04_VALIDATOR_PRIVKEY=$NODE_04_VALIDATOR_PRIVKEY
 NODE_04_P2PIDENTITYPRIVATEKEY=$NODE_04_P2PIDENTITYPRIVATEKEY
-NODE_05_ACCOUNTID=$NODE_05_ACCOUNTID
-NODE_05_VALIDATOR_PRIVKEY=$NODE_05_VALIDATOR_PRIVKEY
 NODE_05_P2PIDENTITYPRIVATEKEY=$NODE_05_P2PIDENTITYPRIVATEKEY
-NODE_05_P2PPUBKEY=$NODE_05_P2PPUBKEY" \
+
+BLOCKISSUER_PRV_KEY=$BLOCKISSUER_PRV_KEY
```

Comment:
Better to rename them with the node they apply to, and move them next to the others of the same node.